### PR TITLE
Update info

### PR DESCRIPTION
**Pull request by cmgustavo**

### CODE REVIEW

File: i18n/docs/updateinfo_en.txt [hunk 1]
```diff
@@ -1,4 +1,7 @@
-* Adds option to set a different Wallet Service per wallet
-* Enhances  CSV generator (only desktop version)
-* Updates Cordova to version 5.3.3
-* Fixes minor bugs
+* TREZOR (for ChromeApp)
```
Comment (matiu) [inline i18n/docs/updateinfo_en.txt]: TREZOR support (only in Chrome App version)

File: i18n/docs/updateinfo_en.txt [hunk 2]
```diff
@@ -1,4 +1,7 @@
...
+* Improves notifications
```
Comment (matiu) [inline i18n/docs/updateinfo_en.txt]: * New notification system. Reduced bandwidth usage and better handling of network disconnections.

File: i18n/docs/updateinfo_en.txt [hunk 3]
```diff
@@ -1,4 +1,7 @@
...
+* Addressbook
+* Transaction history cache and better ordering
+* Fixes BIP44 Gaps
```
Comment (matiu) [inline i18n/docs/updateinfo_en.txt]: Better BIP44 Support: prevent gaps in indexes when scanning wallets and generating addresses.

File: i18n/docs/updateinfo_en.txt [hunk 4]
```diff
@@ -1,4 +1,7 @@
...
+* Reset to default BWS URL
```
Comment (matiu) [inline i18n/docs/updateinfo_en.txt]: Remove this line, and add:
* Per Wallet BWS Setting.

**Comment by cmgustavo** [comment]
Updated. Thanks!

**Pull request merged**